### PR TITLE
Archive unused repo

### PR DESCRIPTION
**Pull request by ryjones**
This repo has not had code contributions since 2019

Signed-off-by: Ry Jones <ry@linux.com>

### CODE REVIEW

**Comment by agunde406** [comment]
I agree with the move to archive this repo. We should get approvals from all Maintainers of this repo before merging:

 - [] Andi Gunderson
 - [] Dan Middleton
 - [] James Mitchell
 - [] Logan Seeley
 - [] Peter Schwarz
 - [] Richard Berg
 - [] Ryan Beck-Buysse
 - [] Shawn Amundson

**Review by agunde406 — APPROVED** [review]

**Review by rbuysse — APPROVED** [review]

**Review by rberg2 — APPROVED** [review]

**Review by ltseeley — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Comment by vaporos** [comment]
@ryjones is it a simple thing to unarchive repos later should there be renewed interest?

**Comment by ryjones** [comment]
> @ryjones is it a simple thing to unarchive repos later should there be renewed interest?

Yes - it's a few clicks. The main reason I want to archive unused repos is I keep getting security alerts on unused repos.

**Review by vaporos — APPROVED** [review]

**Review by jsmitchell — APPROVED** [review]

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**
